### DISCUSSION TITLE
How to enbale maximize mode

### DISCUSSION THREAD

**Discussion by SudhinSudhakaran** [microsoft/react-native-windows#11023] [Q&A] [answered]
I have created an app in react native windows. the alignment of the app is breaking when the app is maximized or expanded by dragging. if I close the app and reopen it will open correctly. my requirement is to disable resizing and always be in maximize mode. what is the solution?

**Comment by chrisglein** [comment, marked as answer]
Similar question to #11050, so I'll include the response for that here:

There isn't going to be any direct JS access to this native functionality (since most of React Native core is from mobile applications where this sort of thing isn't relevant), but when you drop into native code you should be able to do these things. I'd search around for "UWP always maximized" and see what you find. Responses like these for example: 
- https://stackoverflow.com/questions/44273754/how-to-ensure-uwp-app-is-always-full-screen-on-launch
- https://learn.microsoft.com/en-us/answers/questions/498425/can-i-maximize-uwp-window-programmatically.html
- https://stackoverflow.com/questions/35247320/how-to-maximize-a-uwp-window-not-fullscreen
